src/main/index.ts

```diff
@@ -41,7 +41,7 @@ export namespace ObjectMapper {
             converstionFunctionsArray.push(struct);
         });
 
-        let conversionFunctionStructure: ConversionFunctionStructure = converstionFunctionsArray[0];
+        let conversionFunctionStructure: ConversionFunctionStructure = (converstionFunctionsArray.length > 0)? converstionFunctionsArray.pop() : undefined;

```

Comment:
This is the actual fix. The other changes are just updates.

Reply:
@shakilsiraj Can you review this PR and merge it to your base branch if it make sense to you?

Reply:
Refer this ticket for more details https://github.com/shakilsiraj/json-object-mapper/issues/80. If you need any additional details, let me know.

Reply:
Hello. Possible to merge this fix, and make a release please?

Reply:
yeah, we can merge this fix. It has been tested well and we are using this branch in few of our projects. Unfortunately, I don't have access to merge this PR. Can you please help in merge the branch and take a new release?

Reply:
@vapkse Can you merge this branch to master if you have access to merge?

Reply:
I don't have access to merge.

Reply:
Is it @shakilsiraj the one can merge? Do you suggest anybody else if you know?